src/components/header/Header.js

```diff
@@ -0,0 +1,38 @@
+import { render } from '../../shared/dom';
+
+const template= (close, open) =>`
+    <header>
+      <img src="/static/img/logo.png" alt="logo" class="logo">
+      <div class="navBar">
+        <ul id="menu" class="menu">
+          <li>Movie</li>
+          <li>TV Series</li>
+          <li>Search</li>
+          <li>Favorite</li>
+          <button id="exit" class="exit"><li>&times;</li></button>
+        </ul>
+        <div class="btns">
+          <button id="icon" class="icon"><img src="/static/img/menuIcon.png" alt="menu"></button>
```

Comment:
Alt should be meaningfull, it's for accesibility reasons, so it should be something like, toggleMenuIcon
Minor